spec/authorization_spec.rb

```diff
@@ -116,6 +117,39 @@
     it "raises an error with a invalid policy constructor" do
       expect { controller.authorize(wiki, :destroy?) }.to raise_error(Pundit::InvalidConstructorError)
     end
+
+    context "when passed a record with the admin namespace" do
+      it "returns the record when the permission check passes" do
+        allow(controller).to receive(:authorize).with(admin_post, :update?).and_return(admin_post)
+        expect(controller.authorize(admin_post, :update?)).to eq(admin_post)
+      end
```

Comment:
This test doesn't seem to be testing the application code at all? 

`allow(controller).to receive(:authorize)` creates a stub , so `allow(controller).to receive(:authorize)` is only exercising that stub, not actually calling the `authorize` method on the controller code.

Reply:
I appreciate the feedback, @dgmstuart! 

The blog post you linked was a great read, I'll refactor these tests asap.